Src/makefile

```diff
@@ -5,7 +5,8 @@
 # Tool documentation is available at http://spinroot.com
 
 CC?=gcc
-CFLAGS?=-O2 -DNXT -Wall -pedantic
+
+CFLAGS?=-O2 -DNXT -Wall -pedantic -g
```

Comment:
-g generate debug info - is this flag required?

Reply:
Leftover, removing

Reply:
Ok